test/membrane/symbol.spec.js

```diff
@@ -0,0 +1,27 @@
+import createSecureEnvironment from '../../lib/browser-realm.js'
+
+it('blue Symbol class properties are inherited in red environments', () => {
+    const symbol = Symbol.for('method');
+    
+    let successful = false;
+    
+    class Base {
+        [symbol]() {
+            successful = true;
+        }
+    }
+
+    const secureEval = createSecureEnvironment(undefined, { Base, symbol });
```

Comment:
can this come from a local sandboxed module instead of using `createSecureEnvironment`? remember that the goal is to avoid calling that in our tests becasue it is going to change.

Reply:
Not sure I understand what you mean. I need to create a secure environment and pass endowments so that the copyRedDescriptors get triggered for a static red proxy. Can you elaborate more ?